apps/api/.env.example

```
@@ -5,4 +5,5 @@ DISCORD_REDIRECT_URI= #http://localhost:8000/auth/discord/callback
 FRONTEND_HOME_URL= #http://localhost:3000
 JWT_SECRET=
 NODE_ENV=development
-REDIS_URL= #for verification
\ No newline at end of file
+REDIS_URL= #for verificationWYNNVENTORY_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)